core/include/usBundleResource.h

```diff
@@ -28,13 +28,14 @@
 
 #include <ostream>
 #include <vector>
+#include <memory>
 
 US_MSVC_PUSH_DISABLE_WARNING(4099) // type name first seen using 'struct' now seen using 'class'
```

Comment:
I do not remember the details anymore, I just know that there was a reason for it back then. Maybe it was me being stupid. I will follow your suggestion and assume it will compile without warnings on all Visual Studio versions for now.